conanfile.txt

```diff
@@ -0,0 +1,15 @@
+[requires]
+up-core-api/1.5.8
```

Comment:
This is probably a good thing, as from what I've been reading, CCI tends to want to see only other Conan packages used as dependencies, not pulling in git submodules or doing git clones.